src/Core/Types/StructureField.cs

```diff
@@ -50,6 +50,9 @@ public StructureField(int offset, DataType type, string name)
 
         public override string Name { get { if (name == null) return GenerateDefaultName(); return name; }
             set { name = value; } }
+
+        public bool NameSet { get { return name != null; } }
+
```

Comment:
Please rename the property to `IsNameSet`. I like boolean predicates to indicate their predicateness, and sadly C#, unlike scheme, doesn't allow `NameSet?` as a property name.